### PR TITLE
Initial Microsoft.HybridData PR without examples

### PR DESCRIPTION
**Pull request by ghost**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/hybriddatamanager/resource-manager/Microsoft.HybridData/stable/2016-06-01/hybriddata.json [hunk 1]
```diff
@@ -0,0 +1,2493 @@
...
+          "x-ms-enum": {
+            "name": "State",
+            "modelAsString": false
+          }
+        },
+        "datasource": {
+          "description": "The data source associated with the job definition",
+          "type": "string"
+        },
+        "lastmodified": {
```
Comment (nschonni) [inline specification/hybriddatamanager/resource-manager/Microsoft.HybridData/stable/2016-06-01/hybriddata.json]: lastmodified -> lastModified

File: specification/hybriddatamanager/resource-manager/readme.nodejs.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+## Node.js
+
+These settings apply only when `--nodejs` is specified on the command line.
+Please also specify `--node-sdks-folder=<path to root folder of your azure-sdk-for-node clone>`.
+
+``` yaml $(nodejs)
+nodejs:
+  azure-arm: true
+  package-name: azure-arm-storsimple1200series
```
Comment (daschult) [inline specification/hybriddatamanager/resource-manager/readme.nodejs.md]:
```suggestion
  package-name: azure-arm-hybriddatamanager
```

File: specification/hybriddatamanager/resource-manager/readme.typescript.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+## TypeScript
+
+These settings apply only when `--typescript` is specified on the command line.
+Please also specify `--typescript-sdks-folder=<path to root folder of your azure-sdk-for-js clone>`.
+
+``` yaml $(typescript)
+typescript:
+  azure-arm: true
+  package-name: "@azure/hybriddatamanager"
```
Comment (daschult) [inline specification/hybriddatamanager/resource-manager/readme.typescript.md]:
```suggestion
  package-name: "@azure/arm-hybriddatamanager"
```

File: specification/hybriddatamanager/resource-manager/readme.typescript.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+  output-folder: "$(typescript-sdks-folder)/packages/@azure/hybriddatamanager"
```
Comment (daschult) [inline specification/hybriddatamanager/resource-manager/readme.typescript.md]:
```suggestion
  output-folder: "$(typescript-sdks-folder)/packages/@azure/arm-hybriddatamanager"
```

**Review by ghost — CHANGES_REQUESTED** [review]

**Comment by praries880** [comment]
@t-prta kindly fix the CI build breaks and take care of these [linter issues](https://travis-ci.org/Azure/azure-rest-api-specs/jobs/499234664#L818).

**Comment by KrisBash** [comment]
This appears to be a new RP. Please schedule a review meeting with ARM API Review and Azure API Review Board.

**Pull request closed without merging**